lib/coverband/adapters/base.rb

```diff
@@ -76,10 +86,11 @@ def file_hash(file)
       def expand_report(report)
         expanded = {}
         report_time = Time.now.to_i
+        updated_time = self.type == Coverband::Collectors::Coverage::EAGER_TYPE ? nil : report_time
```

Comment:
This is great! I didn't think of the benefit we would get here with tracking last report time.

Reply:
yeah I was really happy to see how easy it was to implement this after you change, makes me think you nailed the breakdown on the data we need. Awesome!